fission/function.go

```diff
@@ -62,6 +65,22 @@ func createArchive(client *client.Client, fileName string) *fission.Archive {
 
 		archive.Type = fission.ArchiveTypeUrl
 		archive.URL = archiveUrl
+
+		f, err := os.Open(fileName)
+		if err != nil {
+			fatal(fmt.Sprintf("Error finding file %v: %v", fileName, err))
```

Comment:
please use checkErr like the rest of the cli -- `checkErr(err, fmt.Sprintf("find file %v", fileName))`